src/main/webapp/app/exam/overview/exercises/quiz/quiz-exam-submission.component.scss

```diff
@@ -1,46 +1,3 @@
-.quiz-navigation {
-    .draganddropcolor-question {
-        background-color: var(--quiz-participation-footer-dnd-background) !important;
-    }
-    .multiplechoicecolor-question {
-        background-color: var(--quiz-participation-footer-mc-background) !important;
-    }
-    .shortanswercolor-question {
-        background-color: var(--quiz-participation-footer-sa-background) !important;
-    }
-    .changed-question {
-        background-color: var(--quiz-participation-footer-changed-background) !important;
-    }
-}
-
-.stepwizardquiz {
-    &__step {
-        .fa {
-            vertical-align: initial;
-        }
-
-        .btn {
-            cursor: help;
-            opacity: 1 !important;
-        }
-    }
-}
-
-.btn-circle {
-    width: 30px;
-    height: 30px;
-    text-align: center;
-    font-size: 12px;
-    font-family: Arial, Helvetica, sans-serif;
-    border-radius: 15px;
-    border-spacing: 5px;
-    padding: 6px 0;
-
-    &.btn-light {
-        border-color: #ffffff;
-    }
-}
-
```

Comment:
This appears to be copied from `quiz-participation.component.scss` and not maintained